README.md

```diff
@@ -40,11 +40,11 @@ appium driver install chromium
 To start an automation session targeting this driver, construct a set of options/capabilities in
 any WebDriver client that (minimally) includes the following:
 
-|Capability|Value|
-|---|---|
-|`platformName`|One of `macOS`, `Linux`, or `Windows` (depending on your system|
-|`browserName`|`browserName` to the running WebDriver process. For example, `chrome` or `chromium` is for chromedriver, `MicrosoftEdge` or `msedge` is for msedge driver.|
-|`appium:automationName`|`Chromium`|
+|Capability| Value                                                                                                                                                      |
+|---|------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
there is no need to change this line, GitHub ill format the table properly

Reply:
same about extra spaces

Reply:
It is my IntelliJ. It does so automatically.

Reply:
Maybe then disable the plugin or use a different editor. It would be painful for someone who is going to change this file afterwards to keep the exact amount of spaces or dashes to preserve the formatting.

Reply:
Changes reverted for readme